scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -130,10 +130,103 @@ def __init__(self):
+
+    ### Node Labels
+
+    The first approach is to add the label `agent.config.scalyr.com/events_leader_candidate=true` to any node that you wish to be eligible to become the events collector.  This can be done with the following command:
+
+    `kubectl label node <nodename> agent.config.scalyr.com/events_leader_candidate=true`
+
+    Where `<nodename>` is the name of the node in question.  You can set this label on multiple nodes, and the agent will use the oldest of them as the event collector leader.  If the leader node is shutdown, it will fallback to the next oldest node and so on.
+
+    Once the labels have been set, you also need to configure the agent to query these labels.  This is done via the `check_labels` configuration option in the Kubernetes Events monitor config (which is off by default):
+
+    ```
+    ...
+    monitors: [
+      {
+        module: "scalyr_agent.builtin_monitors.kubernetes_events_monitor",
+        ...
+        check_labels: true
+      }
+    ]
+    ...
+    ```
+
+    If `check_labels` is true then instead of querying the API server for all nodes on the cluster, the Scalyr agents will only query the API server for nodes with this label set.  In order to reduce load on the API server, it is recommended to only set this label on a small number of nodes.
+
+    This approach reduces the load placed on the API server and also provides a convenient fallback mechanism for when an exister event collector leader node is shutdown.
+
+    ### Hardcoded Event Collector
+
+    The second approach is to manually assign a leader node in the agent config.  This is done via the `leader_node` option of the Kubernetes Events monitor:
+
+    ```
+    ...
+    monitors: [
+      {
+        module: "scalyr_agent.builtin_monitors.kubernetes_events_monitor",
+        ...
+        leader_node: <name of leader node>
+      }
+    ]
+    ...
+    ```
+
+    When the leader node is explicitly set like this, no API queries are made to the K8s API server and only the node with that name will be used to query K8s events.  The downside to this approach is that it is less flexible, especially in the event of the node shutting down unexpectedly.
+
+
+    Since the leader election relies on some assumptions and could impact load on the K8s API server, The leader election algorithm does rely on a few assumptions to work correctly and could, for large clusters, impact performance.
```

Comment:
This paragraph needs to be fixed.

Reply:
Fixed in what way?  Do you mean the extra blank line or is there more explanation needed?

In any case, I've removed the blank line.

Reply:
That paragraph starting with `Since the leader election relies on...` looks like it has two similar sentences that were accidentally joined together.  At the very least, the `T` in `The` should be lower-cased.  Also, there's a lot of repetitiveness to the sentence (repeating of `assumptions`, `impact`, `relies`, etc).

Reply:
Ah sorry, didn't read the full paragraph.  That was just a straight copy/paste from [the existing documentation](https://www.scalyr.com/help/monitors/kubernetes-events).  I've modified it to run nicely in to the next section on disabling the events.